Formula/gtk+.rb

```diff
@@ -48,13 +58,17 @@ class Gtkx < Formula
   end
 
   def install
+    backend = "quartz"
```

Comment:
Can we pull this into separate method and just call it in install method and test block later?